src/pages/apply.js

```diff
@@ -361,7 +361,7 @@ const ApplicationForm = () => {
           <Input
             type="text"
             name="firstName"
-            value={formState.firstName?.value}
```

Comment:
Are we sure we're ok removing these?

Reply:
The entire `formState` object is initiated with `useState` containing a `value` property
<img width="558" alt="image" src="https://user-images.githubusercontent.com/54227730/174688052-a15ec47c-2010-46e3-9269-fde634c89eb9.png">

https://github.com/ethereum/fellowship-program-website/blob/b944d94035f055f072f7ea83e026e84bd39a4fdd/src/pages/apply.js#L130

Don't see a reason we would need this optional. This pattern wasn't used throughout.